tokumx/tox.ini

```diff
@@ -2,11 +2,11 @@
 minversion = 2.0
 basepython = py27
 envlist =
-    py{27,37}
+    py27
 
 [testenv]
 description =
-    py{27,37}: e2e ready
+    py27: e2e ready
```

Comment:
Integration and e2e tests seems to run fine in py2 and py3 . In what context we have an issue ?

Reply:
> passed b/c the current code doesn't hit incompatible paths